src/User/UserRepository.php

```diff
@@ -26,14 +30,59 @@ public function getSpecialistId(int $userId): int
         return (int) $this->getUserProperty($userId, 'user_specialist_id');
     }
 
-    private function getUserProperty(int $userId, string $property): string
+    public function getNick(int $userId): ?string
+    {
+        return $this->getUserProperty($userId, 'user_nick');
+    }
+
+    private function getUserProperty(int $userId, string $property): ?string
     {
-        return $this->createQueryBuilder()
+        $data = $this->createQueryBuilder()
             ->select($property)
             ->from('users')
             ->where('user_id = :userId')
             ->setParameter('userId', $userId)
             ->execute()
             ->fetchOne();
+        return !is_bool($data) ? $data : null;
+    }
+
+    function count(): int
+    {
+        return (int) $this->createQueryBuilder()
+            ->select("COUNT(*)")
+            ->from('users')
+            ->execute()
+            ->fetchOne();
+    }
+
+    function countActiveSessions(int $timeout): int
+    {
+        return (int) $this->createQueryBuilder()
+            ->select('COUNT(*)')
+            ->from('user_sessions')
+            ->where('time_action > ?')
+            ->setParameter(0, time() - $timeout)
```

Comment:
This might just be a personal preference but I'd use name parameters. It makes queries easier to read especially longer queries or queries with multiple parameters

Reply:
Done 3b077a9